Makefile

```diff
@@ -42,8 +42,8 @@ publish-local:
 	export IS_LOCAL_DEVELOPMENT=true; ./gradlew :libandroid-navigation-ui:uploadArchives
 
 graphs:
-	./gradlew :libandroid-navigation:generateDependencyGraphMapboxLibraries
-	./gradlew :libandroid-navigation-ui:generateDependencyGraphMapboxLibraries
+    ./gradlew :libandroid-navigation:generateDependencyGraphMapboxLibraries
+    ./gradlew :libandroid-navigation-ui:generateDependencyGraphMapboxLibraries
```

Comment:
Did you test this? `Makefile` is a bit picky with the formatting 😬 (if it works for you locally scratch this comment).